<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>background</class>
 <widget class="QWidget" name="background">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>471</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>361</width>
    <height>471</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>361</width>
    <height>471</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget
{
    background-color: white;
    border-radius: 20px;
}

QPushButton
{
    color: white;
    background-color: #6357B7;
    border-radius: 15px;
    font: 14pt &quot;Bahnschrift&quot;;
}

QPushButton:Pressed
{
    background-color: #C397E4;
}

QPushButton:disabled
{
    color: #6357B7;
    background-color: white;
    border-radius: 0px;
}

QLineEdit
{
    color: #6357B7;
    background-color: white;
    border : 2px solid #6357B7;
    border-radius: 15px;
    font: 10pt &quot;Bahnschrift&quot;;
    border: 1px solid #6357B7;
    padding: 10px;
}

QLabel
{
    background-color: transparent;
    color: #6357B7;
    font: 12pt &quot;Bahnschrift&quot;;
}</string>
  </property>
  <widget class="QLabel" name="id_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_User">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="pass_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pass">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="login_Button">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Iniciar Sesion</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
